packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -419,55 +412,13 @@ export default mixins(
 					break;
 			}
 		},
-		async createNewWorkflow(): Promise<void> {
-			const result = this.uiStore.stateIsDirty;
-			if (result) {
-				const confirmModal = await this.confirmModal(
-					this.$locale.baseText('generic.unsavedWork.confirmMessage.message'),
-					this.$locale.baseText('generic.unsavedWork.confirmMessage.headline'),
-					'warning',
-					this.$locale.baseText('generic.unsavedWork.confirmMessage.confirmButtonText'),
-					this.$locale.baseText('generic.unsavedWork.confirmMessage.cancelButtonText'),
-					true,
-				);
-				if (confirmModal === MODAL_CONFIRMED) {
-					const saved = await this.saveCurrentWorkflow({}, false);
-					if (saved) await this.settingsStore.fetchPromptsData();
-					if (this.$router.currentRoute.name === VIEWS.NEW_WORKFLOW) {
-						this.$root.$emit('newWorkflow');
-					} else {
-						this.$router.push({ name: VIEWS.NEW_WORKFLOW });
-					}
-					this.$showMessage({
-						title: this.$locale.baseText('mainSidebar.showMessage.handleSelect2.title'),
-						type: 'success',
-					});
-				} else if (confirmModal === MODAL_CANCEL) {
-					this.uiStore.stateIsDirty = false;
-					if (this.$router.currentRoute.name === VIEWS.NEW_WORKFLOW) {
-						this.$root.$emit('newWorkflow');
-					} else {
-						this.workflowsStore.setWorkflowId(PLACEHOLDER_EMPTY_WORKFLOW_ID);
-						this.$router.push({ name: VIEWS.NEW_WORKFLOW });
-					}
-					this.$showMessage({
-						title: this.$locale.baseText('mainSidebar.showMessage.handleSelect2.title'),
-						type: 'success',
-					});
-				} else if (confirmModal === MODAL_CLOSE) {
-					return;
-				}
-			} else {
-				if (this.$router.currentRoute.name !== VIEWS.NEW_WORKFLOW) {
-					this.workflowsStore.setWorkflowId(PLACEHOLDER_EMPTY_WORKFLOW_ID);
-					this.$router.push({ name: VIEWS.NEW_WORKFLOW });
+		goToRoute(route: string | { name: string }) {
```

Comment:
`createNewWorkflow` was an unused leftover?

Reply:
Yep, that's when we had _New workflow_ menu item in the sidebar (before the _Workflows_ page).